python/tk_houdini/ui_generation.py

```diff
@@ -392,6 +484,17 @@ def get_registered_commands(engine):
         commands.append(AppCommand(cmd_name, cmd_details))
     return commands
 
+def get_registered_panels(engine):
+    """Returns a list of AppCommands for the engine's registered panels.
+
+        engine: The engine to return registered panels for
```

Comment:
Would be nice if we could update this to 
```
:param engine: The engine to return registered panels for
```
which is more in line with our (and sphinx's) conventions